packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -256,22 +261,24 @@ export default class TradeStore extends BaseStore {
 
     @action.bound
     async setActiveSymbols() {
-        const { active_symbols, error } = this.should_refresh_active_symbols
-            ? // if SmartCharts has requested active_symbols, we wait for the response
-              await WS.wait('active_symbols')
-            : // else requests new active_symbols
-              await WS.activeSymbols();
-
+        const { active_symbols, error } =
+            // if SmartCharts has requested active_symbols, we wait for the response
+            this.should_refresh_active_symbols
+                ? await WS.wait('active_symbols')
+                : // else requests new active_symbols
+                  await WS.authorized.activeSymbols();
```

Comment:
Can we revert this back to what it was?

Reply:
I think this is the correct flow. Even based on previous comments on this section, we should wait for an already sent request if `this.should_refresh_active_symbols` is true and then send another request `activeSymbols()`

Reply:
Yes, forgive me. I meant moving `this.should_refresh_active_symbols` back to line 264

Reply:
Ah OK sure.